### DISCUSSION TITLE
Django-rq workers won't start on v1.8.3, due rq v2.0 update

### DISCUSSION THREAD

**Discussion by marcusaram** [peering-manager/peering-manager#886] [General]
### Peering Manager version

v1.8.3

### Python version

3.11

### Steps to Reproduce

I did install peering manager using the documentation. But the rq worker won't start with the error:
ImportError: cannot import name 'Connection' from 'rq'

As a Python developer I was already knowing that this has to do with some dependency has updated.
After some searching and looking at the django-rq source code I found out that rq v2.0 is released two weeks ago and has breaking changes. The django-rq package only limits lower version (from) and not to, so pip install will install the latest version of rq since that satisfied the django-rq dependency.

This can be fixed by
```
pip3 uninstall rq
pip3 install rq==1.16.2
```
You can also add rq==1.16.2 to the requirements.txt to fix this.

### Expected Behavior

That the rq worker will start.

### Observed Behavior

ImportError: cannot import name 'Connection' from 'rq'

**Comment by gmazoyer** [comment]
This will be fixed in 1.9. In 1.8 you can define a `local_requirements.txt` file and add `rq<2.0` to fix your local installation. Sadly I won't release a 1.8.4 for this as the main branch as diverged too much for it.

**Comment by gmazoyer** [comment]
Converted to a discussion (and pinned it, as well as added as bookmark in Slack) to warn other people about this.
